nav2_bringup/launch/gz_robot_launch.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2023 Open Source Robotics Foundation
```

Comment:
Rename file to `tb3_gz_robot_launch.py` so its sp. to the TB3

Reply:
https://github.com/ros-planning/navigation2/pull/3634/commits/960fdd764c93588349aff374576da1411d16c37f